README.md

```diff
@@ -58,7 +58,13 @@ Set your Github token to query API without strict rate limiting,
 export GITHUB_API_TOKEN=<your-github-token>
 ```
 
-Create a config file named `config.json`,
+Start script with one of the examples provided
+
+```bash
+python main.py config_samples/lido_dao_sepolia_config.json
```

Comment:
I have `zsh: command not found: python` here, but this is probably my setup only...